docs/pages/config.mdx

```diff
@@ -125,3 +123,64 @@ The global configuration keys are all optional.
   - <a id="upgradeableWorldImplementation" />
     **`upgradeableWorldImplementation`** a `bool`: Whether the `World` is to be deployed behind a proxy to [enable upgrades
     of the core World implementation](/world/upgrades). The default is `false`.
+
+- **`modules`**: a list of modules to be installed into the `World`.
+  Each entry in this list is a record with these fields:
+
+  - **`name`** a `string`: The name of the module.
+  - **`artifactPath`** a `string`: The path to the compiled version of the contract with the module.
+    In our case, [the module](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/modules/keyswithvalue/KeysWithValueModule.sol) is part of MUD's [`@latticexyz/world-modules`](https://github.com/latticexyz/mud/tree/main/packages/world-modules) package.
+  - **`root`** a `bool`: Whether the module is to run with the same permissions as [a `System` in the root namespace](/world/systems#root-systems) or not.
+    Only set to `true` when necessary (for example, to add hooks to a table in a different namspace).
+  - **`args`** a list of arguments to the module, each of which is a record with two keys:
+
+    - **`type`** a string: the Solidity type for the variable.
+    - **`value`** the value of the argument.
+
+    If the argument needs to be a [`ResourceId`](/world/resource-ids), you can create it using either `resourceToHex` or `resolveTableId`.
+
+    <details>
+
+    <summary>Use `resolveTableId` to refer to a table defined in the same configuration file.</summary>
+
+    1. In `mud.config.ts`, import `resolveTableId`.
+
+       ```typescript
+       import { resolveTableId } from "@latticexyz/world/internal";
+       ```
+
+    1. Use this syntax for `args`.
+       The return value is the entire record required for the argument, `type` and `value`.
+
+       ```typescript
+       args: [resolveTableId("app__Tasks")];
+       ```
+
+    </details>
+
+    <details>
+
+    <summary>Use `resourceToHex` to specify the full resource information.</summary>
+
+    1. In `mud.config.ts`, import `resourceToHex`.
+
+       ```typescript
+       import { resourceToHex } from "@latticexyz/world/internal";
```

Comment:
this still lives in common

and I'm not sure we need to document this as I'd rather folks not lean on this particular approach (gives us less flexibility to change the underlying implementation)

Reply:
OK, I'll remove that.